src/lib/components/KudoCard.svelte

```diff
@@ -4,22 +4,58 @@
 	import Svg from './Svg.svelte';
 	import viewport from '../actions/useViewportAction';
 	import type { Kudo } from '@prisma/client';
+	import Icon from "@iconify/svelte";
+	import html2canvas from "html2canvas";
 
 	export let kudo: Kudo;
+	$: kudoId = kudo.id.toString();
 	$: kudoTitle = kudoTitles[kudo.kudoTitle as KudoTitles];
 	$: content = kudo.content;
 	$: to = kudo.to;
 	$: from = kudo.from;
 	$: img = JSON.parse(kudo.img);
 
 	export let animate = false;
+	export let showDownload = true;
```

Comment:
please negate.

call it hideDownload and set it to false.
This way you only need to add `hideDownload` in the KudoCard of the PresenationView:

```
<KudoCard animate={true} {kudo} showDownload={false}/>
```

vs.

```
<KudoCard animate={true} {kudo} hideDownload/>
```

Reply:
and yes... animate can be shortened too :P